src/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ add_library(migraphx
     normalize_attributes.cpp
     normalize_ops.cpp
 )
+configure_file(version.h.in version.h)
```

Comment:
This should be under the migraphx directory: `configure_file(version.h.in include/migraphx/version.h)`.

Reply:
Also the header should be installed, which can be done by updating the `rocm_install_targets` below and adding the new include directory:

```cmake
rocm_install_targets(
  TARGETS migraphx
  INCLUDE
    ${CMAKE_CURRENT_SOURCE_DIR}/include
    ${CMAKE_CURRENT_BINARY_DIR}/include
)
```

Reply:
Fixed